### PR TITLE
up and running with ovirt 3.6 blog post

### PR DESCRIPTION
**Pull request by jasonbrooks**
A howto for running oVirt 3.6 with converged Gluster storage.

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
Please make it clear that this hyperconverged setup is not supported at all.

**Comment by sandrobonazzola** [comment]
@thatdocslady @mykaul +1 on my side, have you got any comments or can we merge?

**Comment by thatdocslady** [comment]
I'm actually waiting to publish this post for a little bit, got Yaniv's DevConf post in the queue first and I would like to space these out.

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
```
Comment (thatdocslady) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: The title is a bit misleading, since the post itself is more advanced than "out of the box" and also includes hosted engine and Gluster. Maybe reword to something like "Up and running with oVirt 3.6 Hosted Engine and Gluster Storage"?

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+author: jbrooks
+date: 2016-02-29 14:00:00 UTC
```
Comment (thatdocslady) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: Please change the date to 2016-03-18 09:00:00 CET so that I can publish this tomorrow :)

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+tags: ovirt, gluster, centos
+published: false
```
Comment (thatdocslady) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: Not sure what the "published" flag means here, we didn't use it in any other posts. I guess in this blog's case, if the PR isn't merged then the blog post won't show up anyway.

**Comment by thatdocslady** [comment]
@jasonbrooks made some minor comments, otherwise we can publish this tomorrow. Thanks so much for the great post!!

**Comment by rharmonson** [comment]
The last URL, http://www.ovirt.org/Community, on the page results with a "Page not found." It is missing '/' and should be "http://www.ovirt.org/community/". I apologize if this has already been identified. Using Firefox ESR 38.5.1.

**Comment by rharmonson** [comment]
https://github.com/oVirt/ovirt-site/pull/104

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md (hunk 1 above)
Comment (jasonbrooks) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: It's the minimum configuration for a complete system that can be maintained without shutting all your VMs off -- it's a somewhat historical title, I've been using the same headline for this series since ovirt 3.0, although, early on, it did describe an all in one install. I'm fine w/ you changing the title if you like.

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md (hunk 3 above)
Comment (jasonbrooks) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: You can merge a post, and if the published flag is set to published: false, it won't be built/deployed. I'll change this.

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md (hunk 2 above)
Comment (jasonbrooks) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: will do

File: source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md (hunk 1 above)
Comment (thatdocslady) [inline source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md]: Oh if this title has street cred, I won't mess with it :)
